embeds/helpEmbed.js

```diff
@@ -0,0 +1,16 @@
+const { RichEmbed } = require('discord.js');
+
+module.exports = (message, conf) => {
+    const embed = new RichEmbed()
+        .setTitle(conf.name)
+        .addField('**Description**', conf.description)
+        .addField('**Usage**', conf.usage);
+
+    if (conf.aliases.length > 0) {
+        embed.addField('**Aliases**', conf.aliases);
+    }
+
+    message.channel.send(embed);
+    return;
```

Comment:
How come `return;` is needed in all of these methods?

Reply:
I forget where I read it but I think it was to make the function exit properly :s